balancer/xds/internal/internal_test.go

```diff
@@ -21,25 +21,40 @@ import (
 	"reflect"
 	"strings"
 	"testing"
+	"unicode"
 
 	"github.com/google/go-cmp/cmp"
 	basepb "google.golang.org/grpc/balancer/xds/internal/proto/envoy/api/v2/core/base"
 )
 
+const ignorePrefix = "XXX_"
+
+func ignore(name string) bool {
+	if !unicode.IsUpper([]rune(name)[0]) {
+		return true
+	}
+	if strings.HasPrefix(name, ignorePrefix) {
```

Comment:
This can be simplified to `return strings.HasPrefix` - I'm surprised staticcheck isn't complaining; I'm sure I've seen it do this before.

Reply:
@dfawley We do not flag this construct if it is preceded by another if statement. Some people prefer the consistency and expandability of writing a list of early returns the way it is done here.

Reply:
@dominikh - Aha, that explains it.  Thanks!